metadata-ingestion/src/datahub/ingestion/source/mlflow.py

```diff
@@ -0,0 +1,87 @@
+from dataclasses import dataclass, field
+from typing import Iterable, List
+
+import mlflow.sklearn
+from datahub.configuration import ConfigModel
+from datahub.configuration.common import AllowDenyPattern
+from datahub.ingestion.api.common import PipelineContext
+from datahub.ingestion.api.source import Source, SourceReport
+from datahub.ingestion.source.metadata_common import MetadataWorkUnit
+from datahub.metadata import MLModelPropertiesClass
+from datahub.metadata.com.linkedin.pegasus2avro.common import VersionTag
+from datahub.metadata.com.linkedin.pegasus2avro.metadata.snapshot import MLModelSnapshot
+from datahub.metadata.com.linkedin.pegasus2avro.mxe import MetadataChangeEvent
+from mlflow.entities import ViewType
+
+
+class MlFlowConfig(ConfigModel):
+    tracking_uri: str
+
+    experiment_pattern: AllowDenyPattern = AllowDenyPattern(deny=["Default"])
+    path_pattern: str = 'model/model.pkl'
```

Comment:
Should `path_pattern` be in the docs?

Reply:
Actually this is deprecated, we opted not to download the model artefacts, which are not enough to compute fairness metrics (features are needed too, and not always available within the mlflow tracking server). We will remove this statement and document the Allow / Deny pattern instead.